components/ChallengeMaterial.tsx

```diff
@@ -19,37 +24,45 @@ type ChallengeQuestionCardProps = {
 }
 
 type ChallengeMaterialProps = {
-  challenges?: Challenge[]
+  userSubmissions: UserSubmission[]
+  challenges: Challenge[]
 }
 
-export const ChallengeTitleCard: React.FC<ChallengeTitleCardProps> = props => {
-  const { currentState: state, active } = props
-  const cardStyles: string[] = []
+export const ChallengeTitleCard: React.FC<ChallengeTitleCardProps> = ({
+  submissionStatus,
+  active,
+  title,
+  id,
+  challengeNum,
+  setCurrentChallenge
+}) => {
+  const cardStyles: string[] = ['challenge-title-card']
   if (active) {
     cardStyles.push('challenge-title-card--active')
   }
-  if (state === 'complete') {
+  if (submissionStatus === 'passed') {
     cardStyles.push('challenge-title-card--done')
   } else {
-    cardStyles.push('shadow-sm', 'border-0', 'challenge-title-card')
+    cardStyles.push('shadow-sm', 'border-0')
   }
 
   return (
     <div
       data-testid="challenge-title"
       className={`card mb-2 ${cardStyles.join(' ')}`}
-      onClick={() => props.setCurrentChallenge(props.id)}
+      onClick={() => setCurrentChallenge(id)}
     >
       <div className="card-body d-flex justify-content-between">
-        <div>{`${props.challengeNum}. ${props.title}`}</div>
-        {state === 'complete' && (
+        <div>{`${challengeNum}. ${title}`}</div>
+        {submissionStatus === 'passed' && (
           <img
             width="25px"
             height="25px"
             src="/curriculumAssets/icons/checkmark.svg"
           />
         )}
-        {state === 'pending' && (
+        {(submissionStatus === 'needMoreWork' ||
+          submissionStatus === 'open') && (
```

Comment:
need more work should be rejected X icon. Do we have that?

Reply:
We do not have this icon!

Reply:
song made it so now we good